dev/CommonStyles/Deprecated_themeresources.xaml

```diff
@@ -0,0 +1,36 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <!-- The resources listed in this file are no longer used by WinUI and have been retained for backward compatibility.
+         They will be removed in a future major WinUI update. -->
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Light">
+            <StaticResource x:Key="TabViewItemIconForeground"                               ResourceKey="SystemControlForegroundBaseMediumBrush" />
```

Comment:
>TabViewItemIconForeground [](start = 35, length = 25)

Can we verify that this change doesn't break apps? Basically that an app that references this resource continues to work after this change. I suspect it does but we should confirm

Reply:
I tested that locally here and and apps still work fine using these resources. Can create a test for this if required.

Reply:
@stmoy would it be possible to make XCG reference one of these resources so that we can validate that it still works when we update the release version?

Reply:
@Felix-Dev I think better than using the xcg would be to add something to the IXMP test app that references these moved resources.

Reply:
@StephenLPeters I'm having troubled adding the IXMP test projects to my VS solution. While I can add the `IXMPTestApp` and `IXMPTestApp.TAEF` projects seemingly fine, adding the shared `IXMPTestApp.Shared` project seems to work not correctly:
![image](https://user-images.githubusercontent.com/1398851/92102797-5b17d580-eddf-11ea-8628-6451a09eeec1.png)

These two top-level folders (3c184...., da296....) should not be in the project and I have no idea why VS just decides to add them. They do not exist on the file system.

I suspect these are causing issues here because while I can build the `IXMPTestApp` project, I cannot *run* it. The run process is terminated with error message
```
The program '[0x5548] IXMPTestApp.exe: Program Trace' has exited with code 0 (0x0).
The program '[0x5548] IXMPTestApp.exe' has exited with code -1073740791 (0xc0000409).
```
before even the Application constructor is called (no breakpoint hit).

Thoughts here?

Reply:
I think @llongley added these projects?

Reply:
@Felix-Dev I think I was mistaken, the IXMPTestApp isn't meant for that... you want the test/MuxControlsReleaseTest

Reply:
@StephenLPeters So I started looking at the MuxControlsReleaseTest (NugetPackageTestApp) and I noticed it is currently using WinUI 2.1 (so pretty outdated version of WinUI). This raises the following question for me: I assume you want to test that these deprecated theme resources still can be referenced perfectly fine by apps *before* accepting this PR into master and then publish a new preview build with this PR included. How would that work though seeing as the NugetPackageTestApp currently seems to be setup to consume *published* WinUI nuget packages?

Reply:
Wow, we really should be updating that test app more regularly... @ranjeshj 

The PR pipeline pubishes a nuget package containing your changes in the artifacts. You can download that and change the test app to reference your local copy to manually confirm it works.  Then I will take a note so that the next time we spin a release I'll make sure it works on the unlisted version of the nuget package before listing it.  Great point, thanks for bringing it up.

Reply:
I just looked at the pipeline definition for Mux-NugetReleaseTest-job and turns out it rewrites the project files to use the nuget package that your build output to the artifacts folder already.  So the Winui 2.1 is more of a place holder value and the pipeline is updating it to the correct value.  Soooo, scratch everything I said before, adding a reference to a deprecated resource should be all you need to do, the pipeline would catch the rest.

Reply:
@Felix-Dev does this make sense? are you unblocked here?

Reply:
@StephenLPeters That sounds good, though I won't be able to compile the MuxControlsReleaseTest app locally since the deprecated TabView resources I would add to it did not yet exist in WinUI 2.1. Could I thus update the placeholder to, say, the newest WinUI 2.4 release which includes these resources?

Other than that, I've been quite busy the past days so a little slow with my responses or PRs for now.

Reply:
I see, these were only added a few months ago. @ranjeshj do you think its okay to update the placeholder nuget package so we can make this change and continue to have the Release Test app build locally? I don't think that would result in any
loss of coverage looking at the script the pipeline calls.  Maybe @kmahone has thoughts too.

Reply:
Yes, update the placeholder version number. As you mentioned, it gets overwritten by the pipeline. Just make sure that the script that updates the version number continue to work - from what I remember it is a fairly crude find-and-replace.

Reply:
Great, I will take a look at updating the placeholder version.

Reply:
here is a link to the script that does the find replace https://github.com/microsoft/microsoft-ui-xaml/blob/545405f0b3e525bd09e181fc298943b07c6cf7e8/build/AzurePipelinesTemplates/MUX-NugetReleaseTest-Job.yml#L94

Reply:
@StephenLPeters  Would something like this look okay for now?

![image](https://user-images.githubusercontent.com/1398851/94190256-2661e000-feac-11ea-8a8b-b4fc6fbf81e6.png)

This is a new page in the MUXControlsReleaseTest apps which for now tests *one* of the now deprecated theme resources (so this single resource test is meant to act as a test for all theme resources in question here).

Reply:
What exactly are we testing through this? What would happen if the resource disappears/breaks, would we be able to notice that through the CI?

Reply:
We are testing here that moving the resources into the new deprecated theme resource file will not break apps. It is my understanding that the CI will then run the tests (launch the app and navigate to the different test pages,...) and that a missing resource will thus be noticed because the app will crash after navigationg to the page consuming said resource.

@StephenLPeters to confirm though.

Reply:
I think that is more than enough, I wouldn't really expect anyone to look at it so as long as the page load the resource I think its sufficient.

Reply:
Added the proposed new page to the MUXControlsTest app now.